packages/manager/apps/hub/src/dashboard/incident/status/status.controller.js

```diff
@@ -45,4 +50,16 @@ export default class StatusController {
       },
     ];
   }
+
+  static hasInstances(service) {
+    return service.productType === 'CLOUD_PROJECT';
```

Comment:
can we store `CLOUD_PROJECT` into a const above the `StatusController` declaration.

Reply:
Same than for feature flipping duplication